src/styles/index.ts

```diff
@@ -4971,13 +4971,17 @@ const styles = (theme: ThemeColors) =>
             flex: 1,
         },
 
-        tripReservationIconContainer: {
-            width: variables.avatarSizeNormal,
-            height: variables.avatarSizeNormal,
+        tripReservationIconContainer: (isSmallIcon: boolean) => ({
+            width: isSmallIcon ? variables.avatarSizeSmallNormal : variables.avatarSizeNormal,
+            height: isSmallIcon ? variables.avatarSizeSmallNormal : variables.avatarSizeNormal,
+            borderRadius: isSmallIcon ? variables.avatarSizeSmallNormal : variables.componentBorderRadiusXLarge,
             backgroundColor: theme.border,
-            borderRadius: variables.componentBorderRadiusXLarge,
             alignItems: 'center',
             justifyContent: 'center',
+        }),
+
+        tripReservationTitleGap: {
+            gap: 2,
```

Comment:
Can you share a quick screenshot of where this is used again? I vaguely recall a quick conversation where we decided that maybe it's better to use 4.

Reply:
![image](https://github.com/Expensify/App/assets/47774969/51e15884-6d17-42ee-b740-ca0ae9d86571)

It's used here. You're right, it should be 4, I've adjusted it :)

Reply:
Sweet, thanks!